collateral/deployment/k8s-without-helm-reference.md

```diff
@@ -0,0 +1,53 @@
+# Deploying Sourcegraph to Kubernetes (k8s) without Helm
+
+## Introduction
+This material is intended to be used as a handout to customers working with the Customer Engineering team. The materials are meant to serve as an overview and resource to aid in deep diving into Sourcegraph's documentation and common deployment tasks.
+
+**Table of Contents:**
+
+- [Introduction](#introduction)
+- [Sourcegraph Setup and Deployment](#sourcegraph-setup-and-deployment)
+	- [Prerequisites](#prerequisites)
+  - [Create your cluster](#create-your-cluster)
+  - [Configure your manifest file](#configure-your-manifest-file)
+  - [Install Sourcegraph](#install-sourcegraph)
+- [Next Steps](#next-steps)
+
+## Sourcegraph Setup and Deployment
+
+### Prerequisites
+- Ensure you have your Sourcegraph Enterprise License
+- Access to deploy Kubernetes clusters
+  - Minimum Kubernetes version: `v1.19` and `kubectl` `v1.19`
+  - Support for Persistent Volumes (SSDs recommended)
+- Determine resource requirements to properly size your instance: [Resource Estimator](https://docs.sourcegraph.com/admin/deploy/resource_estimator)
```

Comment:
I defer to you on how you want to phrase this, but the resource estimator often underestimates resources by about 40%, so this functionally is really "use the resource estimator as a starting-off point but pick values in conjunction with the CE"

Reply:
Hmm, this is a good point actually. I think adding something about the CE works for sure, but in general we should have resources that work. If the resource estimator isn't working (which I think I heard elsewhere it isn't always accurate too), that tool should be removed or improved, right? I think there's a team working to improve the estimator but I'm not sure what the stage is.

How does this sound as a "for now" thing and hopefully in the future we can rely more on the resource estimator, or remove it if it doesn't work properly.

```suggestion
- Determine resource requirements to properly size your instance: The [Resource Estimator](https://docs.sourcegraph.com/admin/deploy/resource_estimator) can give you a baseline, however work with your Sourcegraph Customer Engineer to ensure you are sized appropriately
```

Reply:
Don't disagree! But it is the way it currently is (and has been for at least the last 18 months; the whole thing is a giant 100mb application and probably will just get completely rewritten). I believe Delivery is working on improvements this quarter and you can check with them on status.

Reply:
> Don't disagree! But it is the way it currently is (and has been for at least the last 18 months; the whole thing is a giant 100mb application and probably will just get completely rewritten). I believe Delivery is working on improvements this quarter and you can check with them on status.

@amenne tagging you just for awareness, @jwayman as well. This tool is in the docs and isn't useful in it's current state, to the extent that we have been calling out that working with a CE is going to be a better route. I'm going to keep the link to the. resource estimator and the suggested change above of adding in CE language, but I wanted to make sure this is elevated again that we're saying here that our official docs aren't sufficient without CE involvement.